scene/gui/slider.cpp

```diff
@@ -184,10 +184,10 @@ void Slider::_notification(int p_what) {
 					focus->draw(ci,Rect2i(Point2i(),Size2i(style->get_minimum_size().width+style->get_center_size().width,size.height)));
 				*/
 				if (ticks > 1) {
-					int tickarea = size.height - tick->get_height();
+					int grabber_offset = (grabber->get_size().height/2 - tick->get_height()/2);
```

Comment:
Spaces surrounding operators are mandatory. At `/`. Same in Line 208.

Also it seems like your Git email is not the same as your Github's, so it doesn't link to your account. Is this intended?

Reply:
@Noshyaar done! I also linked my email to my github account, didn't notice it wasn't done yet, thanks.